snowplow-tracker.gemspec

```diff
@@ -33,6 +33,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.1'
 
   s.add_runtime_dependency 'contracts', '~> 0.7', '< 0.17'
-  s.add_development_dependency 'rspec', '~> 2.14.1'
-  s.add_development_dependency 'webmock', '~> 1.17.4'
+  s.add_development_dependency 'rspec', '>= 2.0'
+  s.add_development_dependency 'webmock', '>= 1.17', '< 3.9'
```

Comment:
This change has crept into this commit. Might end up with a conflict when merge/cherrypicking, just a heads up. Potentially merge the loosening PR into release after cherrypicking this one will work around it.